website/snippets/_cloud-environments-info.md

```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
```

Comment:
i've read and been told that we should avoid using directional language like below or above for accessibility reasons
```suggestion
The following are situations when dbt Cloud uses a cached copy:
```

Reply:
good flag!